<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolarusGui::MainWindow</class>
 <widget class="QMainWindow" name="SolarusGui::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solarus</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="console_splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="vertical_layout">
        <item>
         <widget class="SolarusGui::QuestsView" name="quests_view"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontal_layout">
          <item>
           <spacer name="horizontal_spacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="play_button">
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontal_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="SolarusGui::Console" name="console" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_quest">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_add_quest"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu_quest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_add_quest">
   <property name="text">
    <string>Add quest...</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SolarusGui::Console</class>
   <extends>QWidget</extends>
   <header>solarus/gui/console.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SolarusGui::QuestsView</class>
   <extends>QListView</extends>
   <header>solarus/gui/quests_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
